packages/amplify-cli/package.json

```diff
@@ -58,6 +58,7 @@
     "@aws-amplify/amplify-frontend-ios": "3.7.6",
     "@aws-amplify/amplify-frontend-javascript": "3.10.17",
     "@aws-amplify/amplify-go-function-template-provider": "1.4.8",
+    "@aws-amplify/amplify-migration": "^0.0.1",
```

Comment:
Consider naming this `@aws-amplify/amplify-gen1-gen2-migration` or something more specific.

Why?
1. There are still folks on very old Gen1 CLI who want to move to latest gen1 and may hope this is what they're looking for.
2. Some day there might be Gen 3.

Reply:
Ack. I have this addressed locally and will include the change in the follow-up PR so it can be properly reviewed.